java-frontend/src/test/java/org/sonar/java/viewer/EGViewer.java

```diff
@@ -42,6 +42,7 @@
 
   private static final ActionParser<Tree> PARSER = JavaParser.createParser(StandardCharsets.UTF_8);
   private final Viewer viewer;
+  private static final boolean SHOW_CACHE = true;
```

Comment:
Naming! notion of cache should probably not appear anywhere except for the walker : this is implementation detail.